packages/unocss/scripts/copy-files.ts

```diff
@@ -0,0 +1,11 @@
+import { copyFileSync } from 'fs'
+import { resolve } from 'path'
+
+// relative to scripts directory
```

Comment:
Could we move this to the top level so we copy them once on build

Reply:
sure, to root script directory